thirdparty/scripts/build_modin.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -x
+
+# Cause the script to exit if a single command fails.
+set -e
+
+TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)/../
+
+# setup env
+if [[ -z  "$1" ]]; then
+  PYTHON_EXECUTABLE=`which python`
+else
+  PYTHON_EXECUTABLE=$1
+fi
+echo "Using Python executable $PYTHON_EXECUTABLE."
+
+$PYTHON_EXECUTABLE -m pip install modin --no-deps --target $TP_DIR/../python/ray/modin_files
```

Comment:
modin should be pinned to a specific commit from the modin repo, so we can make sure that everything is tested appropriately (similar to how we are handling arrow)

Reply:
I am worried about supporting two different versions. This was the simplest solution to support the most recent release of Modin. We test these heavily before releasing.

This was my motivation for doing it this way. I would like to avoid having a different commit for the Ray version and the PyPI release. Let me know what you think of this line of thinking.